### PR TITLE
i18n: global locale file `root` fix

### PR DESCRIPTION
**Pull request by petebacondarwin**
Blocked on https://github.com/angular/angular/pull/33682, which itself is blocked on https://github.com/angular/angular/pull/33699

### CODE REVIEW

File: packages/common/locales/global/ar-IQ.js [hunk 1]
```diff
@@ -9,142 +9,21 @@
...
-      u,
-      [
-        ['03:00', '06:00'], ['06:00', '12:00'], ['12:00', '13:00'], ['13:00', '18:00'],
-        ['18:00', '24:00'], ['00:00', '01:00'], ['01:00', '03:00']
-      ]
-    ]
-  ];
-})(typeof globalThis !== 'undefined' && globalThis || typeof global !== 'undefined' && global ||
-   typeof window !== 'undefined' && window);
+(
```
Comment (josephperrott) [inline packages/common/locales/global/ar-IQ.js]: Did this file fail to clang format?

**Review by josephperrott — APPROVED** [review]
LGTM, just curious about the formatting on a few files?  Did clang run as expected?

File: packages/common/locales/global/ar-IQ.js (hunk 1 above)
Comment (petebacondarwin) [reply packages/common/locales/global/ar-IQ.js]: It said that it ran.

Comment (petebacondarwin) [reply packages/common/locales/global/ar-IQ.js]: It definitely formatted it, since the unformatted version is all on one line. 🤷‍♂

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
- [VE Presubmit](http://test/OCL:279138338:BASE:279138363:1573157145280:704759c1)
- [Ivy Presubmit](http://test/OCL:279138338:BASE:279138706:1573157232808:704e3f6a)

**Comment by AndrewKushnir** [comment]
@petebacondarwin FYI, VE and Ivy Presubmits are successful, but it looks like there is a conflict with master branch now (so I've added a "cleanup" label). Could you plz rebase and fix conflicts? Thank you.

**Comment by petebacondarwin** [comment]
We should land #33682 before this one.

**Comment by petebacondarwin** [comment]
Rebased and ready for presubmit @kara

**Comment by kara** [comment]
[VE presubmit](https://test.corp.google.com/ui#id=OCL:279994608:BASE:279994672:1573581854947:4fbf4b70)

[Ivy presubmit](http://test/OCL:279994608:BASE:279995700:1573582111840:4f6ad176)

**Pull request closed without merging**